src/pages/docs/tenants/index.md

```diff
@@ -10,100 +10,76 @@ navOrder: 60
 hideInThisSection: true
 ---
 
-Most people who use Octopus use it to deploy projects to one or more environment.
+Tenants in Octopus help you deploy software to many production instances, targets, or customers without duplicating effort. This includes:
 
-What happens when you are providing Software as a Service (SaaS) applications and you need to deploy multiple instances of the application for each of your customers? 
+- Delivering [Software as a Service (SaaS)](/use-case/tenanted-deployments/saas/) applications where each customer has its own resources
```

Comment:
For off-docs links, we use full URLs as these pages aren't available on the test environment.

```suggestion
- Delivering [Software as a Service (SaaS)](https://octopus.com/use-case/tenanted-deployments/saas/) applications where each customer has its own resources
```